migrating_to_ams.md

```diff
@@ -0,0 +1,50 @@
+# Migrating from using EGI ActiveMQ Message Brokers to using EGI ARGO Messaging Service
+
+Migration requires upgrading to SSM-X.X.X and adding new values to your configuration.
+
+## Sender
+
+The sender configuration is usually found under `/etc/apel/sender.cfg`. Follow the steps below to migrate.
+
+1. Comment out `bdii` and `network`.
+1. Uncomment `host` and set it to `msg-devel.argo.grnet.gr`.
+1. Add the following as a new section at the top of your configuration:
+```
+[sender]
+# Either 'STOMP' for STOMP message brokers or 'AMS' for Argo Messaging Service
+protocol: STOMP
```

Comment:
this should be `protocol: AMS`, as the instruction says:
> Add the following as a new section at the top of your configuration:

Reply:
Good spot.

Reply:
Sorted.